integrations/kml-parser/src/main/java/ch/stayhome/integrations/kml/job/StayHomeEntryWriter.java

```diff
@@ -0,0 +1,43 @@
+package ch.stayhome.integrations.kml.job;
+
+import java.util.List;
+import java.util.Map;
+
+import feign.Feign;
+import feign.Logger;
+import feign.Retryer;
+import feign.gson.GsonEncoder;
+import lombok.extern.slf4j.Slf4j;
+
+import org.springframework.batch.item.ItemWriter;
+
+@Slf4j
+public class StayHomeEntryWriter implements ItemWriter<StayHomeEntry> {
+
+	private final StayHomeApi api;
+
+	StayHomeEntryWriter(String url) {
+		this.api = Feign.builder()
+				.encoder(new GsonEncoder())
+				.retryer(new Retryer.Default())
+				.logLevel(Logger.Level.FULL)
```

Comment:
we might want to move this into application.yml and reduce logging in production.